AgentInterfaces/VmAgentSettings.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Azure.Gaming.AgentInterfaces
+{
+    public class VmAgentSettings
+    {
+        public bool ProcessCrashDumps { get; set; }
```

Comment:
Boolean should be a question, what about ShouldEnableTelemetry and ShouldEnableCrashDumps?

Reply:
changed to `EnableCrashDumpProcessing` and `EnableTelemetry`